controllers/details.js

```diff
@@ -1,5 +1,25 @@
 'use strict';
 
+var _ = require('underscore');
+var Model = require('../models');
+
 module.exports = function renderDetails(req, res) {
-  res.render('pages/details');
+  var model = new Model(req.session.model);
+  var records = model.get('records');
+  var record = records[0];
```

Comment:
Do we need this? We should always get a sysnum param through? Amend to just
var record;
?